runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1928,26 +1928,45 @@ sub getWorkloadStatsSummary {
 	my $overallAvgRT = 0;
 
 	my $numPassingAi = 0;
+	my $numTotalAi = 0;
 	my $appInstancesRef = $self->workload->appInstancesRef;
+	$numTotalAi = scalar @$appInstancesRef;
+
 	foreach my $appInstanceRef (@$appInstancesRef) {
 		my $appInstanceNum = $appInstanceRef->instanceNum;
 		if ($self->passAll->{$appInstanceNum}) {
 			$numPassingAi++;
+		}
+		# Summary fields are computed across all AIs, both passing and failing
+		{
+			# Uninitialized warnings can occur if this is a failed findmax run. Summary fields will be output as "NA"
+			no warnings 'uninitialized';
 			$totalUsers   += $self->maxPassUsers->{$appInstanceNum};
 			$opsSec       += $self->opsSec->{$appInstanceNum};
 			$httpReqSec   += $self->reqSec->{$appInstanceNum};
 			$overallAvgRT += $self->overallAvgRT->{$appInstanceNum};
 		}
 	}
-	
-	$csvRef->{"users-total"}       = $totalUsers;
-	$csvRef->{"opsSec-total"}       = $opsSec;
-	$csvRef->{"httpReqSec-total"}   = $httpReqSec;
-	if ($numPassingAi > 0) {
-		$csvRef->{"overallAvgRT-total"} = $overallAvgRT / $numPassingAi;
+			
+	if ($numPassingAi == $numTotalAi) { # If all AIs pass, the run passes
+		$csvRef->{"Pass"}				= "Y";
 	} else {
-		$csvRef->{"overallAvgRT-total"} = 0;		
+		$csvRef->{"Pass"}				= "N";
 	}
+
+	# If this is a failed findMax run, print "NA" for summary fields
+	if ( ($csvRef->{"Pass"} eq "N") && ($self->workload->appInstancesRef->[0]->getParamValue('loadPathType') eq "findmax") ) {
```

Comment:
Should it check for syncedfindmax also?

Reply:
Yes.  You need to check for syncedfindmax here as well.  I don't think that loadPathType existed in the code you were using originally.

Reply:
Thank you for the catch. This has been fixed in commit ebc4db1.